config/initializers/slack_api.rb

```diff
@@ -4,12 +4,12 @@
   config.token = ENV['slack_api_token']
 end
 
-#client = Slack.realtime
+client = Slack.realtime
 
-#client.on :team_join do |data|
-#  welcome = "Welcome to slashrocket, <@#{data['name']}>! Type `/rocket` for a quick tour of our channels. :simple_smile:"
-#  options = { channel: '#general', text: welcome, as_user: true }
-#  Slack.chat_postMessage(options)
-#end
+client.on :team_join do |data|
+  welcome = "Welcome to slashrocket, <@#{data['user']['name']}>! Type `/rocket` for a quick tour of our channels. :simple_smile:"
+  options = { channel: '#general', text: welcome, as_user: true }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.